introspection_response_writer.go

```diff
@@ -204,35 +204,50 @@ func (f *Fosite) WriteIntrospectionResponse(rw http.ResponseWriter, r Introspect
 		return
 	}
 
-	expiresAt := int64(0)
+	response := map[string]interface{}{
+		"active": true,
+	}
+
+	extraClaimsSession, ok := r.GetAccessRequester().GetSession().(ExtraClaimsSession)
+	if ok {
+		extraClaims := extraClaimsSession.GetExtraClaims()
+		if extraClaims != nil {
+			for name, value := range extraClaims {
+				switch name {
+				// We do not allow these to be set through extra claims.
+				case "exp", "client_id", "scope", "iat", "sub", "aud", "username":
```

Comment:
Must include `iss`, `jti`, `nbf`

Reply:
This is not really possible to limit. Because currently regular claims from JWT tokens are passed through to introspection endpoint using `GetExtraClaims`. I tried to not do it like that, but I get import cycle. See: https://github.com/ory/fosite/pull/579/commits/04cd6255188941af972552da984f65f9cf6cde7e